### DISCUSSION TITLE
Add function that move camera position to the place where the mesh is in the fructsum of camera

### DISCUSSION THREAD

**Discussion by tkoyama010** [pyvista/pyvista#1419] [Ideas]
As I answered in  https://github.com/pyvista/pyvista/discussions/1417  we cannot fit camera position to the place where the mesh is in the fructsum of camera automatically.
It must be useful, if we add function that move camera position to the place where the mesh is in the frustum of camera

![camera_1_0](https://user-images.githubusercontent.com/7513610/115476330-9be43200-a27c-11eb-9b8b-4d7363f52dee.png)

**Comment by akaszynski** [comment]
Did we implement this already?
https://github.com/pyvista/pyvista/blob/e54d34df834929f60662d1789d436d675255e191/pyvista/plotting/plotting.py#L435-L439
